starter_code/models/Movie.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose')
+const Schema = mongoose.Schema
+
+const movieSchema = new Schema({
+    title: String,
+    director: String,
+    stars: Array,
+    image: String,
+    description: String,
+    showtimes: Array,
+})
+
```

Comment:
Array here leaves a lot of information to be specified. It would be better if you added more information. Array of strings would be better :) More specific.

Reply:
You mean to make sure only strings can be added to the array `showTimes`?